### ISSUE TITLE
JavaDoc: UnusedImports check fails for inline @link tags in JavaDoc comments if linebreak is present

### ISSUE THREAD

**Issue by tsjensen** [checkstyle/checkstyle#359]
This is an issue that was already raised in #88, so you may want to just reopen #88. The problem occurs if the `{@link Qax}` tag contains a line break. This happens often in IntelliJ IDEA, which features a rather brutal Javadoc formatter.

_The following description is mostly copied from #88, but adapted for the line break case:_

The _UnusedImports_ check reports a violation in case an imported type is only referenced from within an inline `@link` tag (also if _processJavadoc_ is set to `true`), if that tag contains a line break:

``` java
import com.example.Qax;

/**
 * @param T The bar. May be a rather longish description which placed the {@link
 * Qax} at the end of a line.
 */
public class Foo<T> {};
```
##

**Comment by silentj** [comment]
This looks like a duplicate of #2840. The example is slightly different but the fix is the same for both. #2840 has much more discussion so I'd suggest closing this issue.

**Comment by tsjensen** [comment]
Let's close it when it's fixed, ok? Like Roman said above, we should be sure that the new implementation supports this use case.

**Comment by romani** [comment]
closed in favor of #2840.
